apps/server/src/routers/api/v1/user.rs

```diff
@@ -454,26 +454,24 @@ async fn create_user(
 			let permissions = input
 				.permissions
 				.into_iter()
-				.filter_map(|p| {
-					let p_str = p.to_string();
-					if p_str.is_empty() {
-						None
-					} else {
-						Some(p_str)
-					}
-				})
+				.map(|p| p.to_string().trim().to_owned())
+				.filter(|p| !p.is_empty())
 				.collect::<Vec<String>>();
 
 			let created_user = client
 				.user()
 				.create(
 					input.username.to_owned(),
 					hashed_password,
-					vec![
-						user::is_server_owner::set(false),
-						user::permissions::set(Some(permissions.join(","))),
```

Comment:
This line was responsible for adding an empty string when no permissions were provided during user creation